Add Materials Page to docs (#7642)
prometheus-operator/prometheus-operator: Documentation/Materials.md

---
title: Materials
draft: false
---

> This page contains all the presentations and blogs that demonstrate and explain about the concepts and examples of Prometheus-Operator.

Reviewer: (nit) just to make it a bit shorter?
```suggestion
This page contains presentations and blog articles that talk about the Prometheus Operator.
```

> If you have made a presentation or blog that demonstrated or referenced Prometheus-Operator, please open a PR to add it to this page!

## Presentations

> * [Optimizing Scrape Configs with Prometheus Operator](https://youtu.be/Yd0kuAHaWjE?si=fta8nhRhCH5NRatY) - Nicolas Takashi - PromCon 2024 Lightning Talk

Reviewer: ```suggestion
* [Optimizing Scrape Configs with Prometheus Operator](https://youtu.be/Yd0kuAHaWjE?si=fta8nhRhCH5NRatY) - Nicolas Takashi - PromCon 2024 (lightning talk)
```

* [Harnessing the Potential of Prometheus Agent Mode](https://youtu.be/wkkXh8X0N8s?si=PVqsvzxM5O0sfvMd) - M Viswanath Sai and S Ashwin - PromCon 2024

* [Architecting Growth: Scaling Tactics for Prometheus Metrics Collection](https://youtu.be/bVICOulH5IY?si=M1GKX0lDmuBzyCzH) - Arthur Silva Sens & Nicolas Takashi - KubeCon + CloudNativeCon North America 2024

> * [ScrapeConfig CRD: The Rising Star in the Prometheus-Operator Ecosystem](https://youtu.be/RabXsaOl1TE?si=Sc82jMpIgddcR212) - Jayapriya Pai - KCD Kerala

Reviewer: 2024?

Author: Yeah missed that :+1:

* [Prometheus-Operator: What's next?](https://www.youtube.com/live/ymR57Q0qqg4?si=JUmYjkfSOJe4qpam&t=27386) - Simon Pasquier - PromCon 2023

* [Deep Dive into the Prometheus Operator Ecosystem](https://youtu.be/P__R4CFFxEQ?si=1YPkY-X-4ST4YJ7j) - Jayapriya Pai - KCD Bengaluru 2023

* [How to Be 10x SRE? A Deep Dive to Prometheus Operator](https://youtu.be/Uph_Say4D3M?si=Z3mmdBMWjA9PjBt2) - Jayapriya Pai & Haoyu Sun - Prometheus Day Europe 2022

* [Monitoring Kubernetes with Prometheus-Operator](https://youtu.be/MuHPMXCGiLc?si=2tpfuyMzRVh0MR8-) - Lili Cosic - Cloud Native Computing Berlin Meetup 2021

> ## Blogs

Reviewer: What would be the order when adding? I would suggest new entries are added at the bottom.

Author: I think having the latest talks on top might be better. We can probably add an instruction on how to add your talk.
